src/styles/components.css

```
@@ -111,3 +111,32 @@
+/* ===== EMAIL HTML RENDER (run h538172) =====
+   Contém o layout de e-mails HTML dentro do bubble do chat:
+   tabelas de 600px+, imagens e words longas não podem estourar o container. */
+.email-html-body {
+  color-scheme: light;
+  color: #111;
+  background: #fff;
+  line-height: 1.5;
+  white-space: normal;
+  display: flow-root;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)